cmd/keymasterd/main.go

```diff
@@ -2411,7 +2415,6 @@ func main() {
 	}
 
 	serviceTLSConfig := cfg
-	serviceTLSConfig.ClientAuth = tls.RequestClientCert
```

Comment:
As discussed, this would lead to a confusing prompt for users of the previous version(s) of Safari.